### PR TITLE
NICAB location updates (Feb. 16)

### PR DESCRIPTION
**Pull request by jiannis**
See spreadsheet titled _"08022016 Bureaux Addresses and Telephone Numbers for HMT"_
- 1 location added
- 15 locations amended

### CODE REVIEW

File: public/js/nicab.json [hunk 1]
```diff
@@ -73,8 +73,8 @@
   },
   {
     "title": "Coleraine",
-    "address": "1-5 Brook Street\nColeraine\nLondonderry\nBT52 1PW",
-    "phone": "028 7032 9775",
+    "address": "24 Lodge Road\nColeraine\nBT52 1NB",
```
Comment (andrewgarner) [inline public/js/nicab.json:76]: This will need the long and lat re-calculated.

File: public/js/nicab.json [hunk 2]
```diff
@@ -82,32 +82,41 @@
   },
   {
     "title": "Cookstown",
-    "address": "7 - 11 William Street\nCookstown\nCo. Tyrone\nBT80 8AX",
-    "phone": "028 8676 6891",
+    "address": "The Enterprise Centre\nDerryloran Industrial Estate, Sandholes\nCookstown\nBT80 9LU",
```
Comment (andrewgarner) [inline public/js/nicab.json]: "Derryloran Industrial Estate\nSandholes" not "Derryloran Industrial Estate, Sandhogs"

File: public/js/nicab.json [hunk 3]
```diff
@@ -82,32 +82,41 @@
...
-    "phone": "028 7134 7047",
+    "phone": "028 7136 2444",
     "hours": "Monday to Friday, 9am to 1pm and 2pm to 5pm",
     "lat": 54.99967789999999,
     "lng": -7.3219594,
     "id": "03ca70f3-f76a-4563-a243-571000f80ab4"
   },
   {
+    "title": "Derry Water Side",
+    "address": "Unit 1 Spencer House\n14-22 Spencer Road\nDerry\nBT47 6QA",
```
Comment (andrewgarner) [inline public/js/nicab.json]: "Unit 1\nSpencer House\n14-22 Spencer Road\nDerry\nBT47 6QA"

File: public/js/nicab.json (hunk 1 above)
Comment (jiannis) [inline public/js/nicab.json:76]: I was getting the same coordinates..

``` ruby
require 'geocoder'

address = '24 Lodge Road, Coleraine, BT52 1NB'
coordinates = Geocoder.coordinates(address)
=> [55.1315202, -6.6656869]
Geocoder.search(coordinates.join(",").first.address
=> "24 Lodge Rd, Coleraine, Coleraine BT52 1NB, UK"
```

The new address is [here in Google Earth](https://www.google.co.uk/maps/place/55°07'53.5%22N+6°39'56.5%22W/@55.1305355,-6.6686615,520m/data=!3m1!1e3!4m2!3m1!1s0x0:0x0) and the [old address](https://www.google.co.uk/maps/place/5+Brook+St,+Coleraine+BT52+1EN/@55.132578,-6.6671981,501m/data=!3m1!1e3!4m2!3m1!1s0x486024b55c1129e5:0xe4dc7c7d7041531a) is a couple of blocks away. The old address is @ 55.133378, -6.6671518 according to Google's geocoder. No idea why we had the old address but the new coordinates..

Comment (andrewgarner) [inline public/js/nicab.json:76]: Bizarre! :confused:

**Pull request merged**